### PR TITLE
add: `teams-for-linux-deb` 

### PR DESCRIPTION
**Pull request by Zahrun**
## Progress
* [x]  Edit `packagelist`
* [x]  Add initial pacscript  (maintainer-less)
* [x]  Contact devs (https://github.com/IsmaelMartinez/teams-for-linux/issues/510)
* [x]  Add `maintainer` to pacscript: resolved to @Zahrun (https://github.com/IsmaelMartinez/teams-for-linux/issues/510#issuecomment-1004643037)

### CODE REVIEW

**Comment by wizard-28** [comment]
@Henryws, I don't think we could do the procedures (contacting dev) here. So we shall skip them right?

File: packages/teams-for-linux-deb/teams-for-linux-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name="teams-for-linux-deb"
+pkgname="teams-for-linux"
+version="1.0.20"
+url="https://github.com/IsmaelMartinez/teams-for-linux/releases/download/v${version}/teams-for-linux_${version}_amd64.deb"
+breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app ${pkgname}-deb"
+gives="${pkgname}"
+description="Unofficial Microsoft Teams client for Linux using Electron. It uses the Web App and wraps it as a standalone application using Electron."
+hash="a244a41defd8f1fb35da6540414efdb02b9a7ade89487bc5dff20fb1f45a988e"
+maintainer="Zahrun@github.com"
```
Comment (wizard-28) [inline packages/teams-for-linux-deb/teams-for-linux-deb.pacscript:9]:
```suggestion
maintainer="Zahrun <Zahrun@github.com>"
```

File: packages/teams-for-linux-deb/teams-for-linux-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name="teams-for-linux-deb"
+pkgname="teams-for-linux"
+version="1.0.20"
+url="https://github.com/IsmaelMartinez/teams-for-linux/releases/download/v${version}/teams-for-linux_${version}_amd64.deb"
+breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app ${pkgname}-deb"
```
Comment (wizard-28) [inline packages/teams-for-linux-deb/teams-for-linux-deb.pacscript:5]:
```suggestion
breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Follow the new PR guidelines (it's has not been added to `CONTRIBUTING.md` yet). See #558, #567 for examples.

**Comment by wizard-28** [comment]
Okay, so now we wait max. 1 week for the dev to reply, if they don't within that time frame, then we merge

**Review by wizard-28 — CHANGES_REQUESTED** [review]
@Zahrun, add yourself as the maintainer

File: packages/teams-for-linux-deb/teams-for-linux-deb.pacscript (hunk 2 above)
Comment (wizard-28) [inline packages/teams-for-linux-deb/teams-for-linux-deb.pacscript:5]:
```suggestion
breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by Zahrun** [comment]
Do we have a color to use with a pacstall badge like AUR has blue?
[AUR: teams-for-linux](https://aur.archlinux.org/packages/teams-for-linux)
https://shields.io/

**Comment by wizard-28** [comment]
> Do we have a color to use with a pacstall badge like AUR has blue? [AUR: teams-for-linux](https://aur.archlinux.org/packages/teams-for-linux) https://shields.io/

Use `#00958C`

**Comment by Zahrun** [comment]
Nice! 
[Pacstall: teams-for-linux-deb](https://github.com/pacstall/pacstall-programs/tree/master/packages/teams-for-linux-deb)

**Comment by wizard-28** [comment]
> Nice! [Pacstall: teams-for-linux-deb](https://github.com/pacstall/pacstall-programs/tree/master/packages/teams-for-linux-deb)

So, I should merge this first right?

**Comment by Zahrun** [comment]
Yes, so that the link will direct to the package

**Pull request merged**